tests/test_reporter.py

```diff
@@ -99,3 +124,171 @@ def test_match_three_traces(self):
             msg = ("Trace_{} doesn't match.\n"
                    "{!r} != {!r}").format(idx, trace, expected_traces[idx])
             self.assertEqual(trace, expected_traces[idx], msg=msg)
+
+
+class TestReporter(unittest.TestCase):
+    """Test cases for Reporter class."""
+
+    def setUp(self):
+        """Set up test fixtures."""
+        # Write a kernel .config file
+        self.tmpdir = tempfile.mkdtemp()
+        self.tempconfig = "{}/.config".format(self.tmpdir)
```

Comment:
Can we have the config created only if the merge passed, or would it be too complicated? That way we can check merge failures don't require it for reports. Eventually, we should be grabbing config only if the build was attempted, but we need to implement that first.

Reply:
I'll just delete the temporary config file before I test for the merge failure.